### ISSUE TITLE
\* in install instructions does not work

### ISSUE THREAD

**Issue by QuLogic** [fedora-infra/bodhi#3039]
Looking at [this update](https://bodhi.fedoraproject.org/updates/FEDORA-2019-050d9fdc17), the install instructions are:
> sudo dnf install --enablerepo=updates-testing --advisory=FEDORA-2019-050d9fdc17 \*

However, this does not work:
```
$ sudo dnf install --enablerepo=updates-testing --advisory=FEDORA-2019-050d9fdc17 \*
Last metadata expiration check: 0:29:58 ago on Sun 24 Feb 2019 09:40:21 PM EST.
No security updates for argument "*"
No match for argument: *
  * Maybe you meant: 3dprinter-udev-rules
Error: Unable to find a match
```

**Comment by mattiaverga** [comment]
I think that's because the update has not yet reached the mirrors. In fact I can reproduce the same output by using the most recent newpackage updates, while with older updates it just works:
```
$ sudo dnf install --enablerepo=updates-testing --advisory=FEDORA-2019-51892576be \*
Ultima verifica della scadenza dei metadati: 0:01:04 fa il Tue Feb 26 09:37:00 2019.
No security updates for argument "*"
Nessuna corrispondenza per l'argomento: *
  * Forse si intende: CharLS
Errore: Impossibile trovare una corrispondenza
```
This also happens for other upgrade types:
```
$ sudo dnf upgrade --enablerepo=updates-testing --advisory=FEDORA-2019-7462acf8ba
Ultima verifica della scadenza dei metadati: 0:11:17 fa il Tue Feb 26 09:37:00 2019.
Nessun aggiornamento di sicurezza richiesto, ma sono disponibili 303 aggiornamenti
Dipendenze risolte.
Nessuna operazione da compiere.
Fatto!
```
Unfortunately, I have no idea how to avoid displaying the update command until we are sure the update hits the mirrors. At the moment we just choose to do that when the update changes its status to "testing", which for Bodhi means it has been pushed to the mirrors... the only two solutions I can think of are: show the command after a predefined delay of the datetime the update had been pushed to testing; or show a warning sentence after the command box to inform user that it can be a delay before the command actually works.
@bowlofeggs that do you think?

**Comment by bowlofeggs** [comment]
Yeah there isn't a good way for Bodhi to know which mirrors have the update when. Keep in mind that mirrors lag by widely varying amounts too. Fedora's mirror system doesn't pull a mirror out of rotation unless it is more than three update pushes behind (which typically means they are considered "up to date" if they have data within the past three days or so, since we typically push daily.)

There's actually another potential issue I've observed, where dnf invalidates its cache on one run but doesn't redownload repository metadata so says there is nothing to do. Then on an immediate second run it downloads the data and sees updates. I've noticed this particularly often when using ```--enablerepo updates-testing```. I haven't searched to see if this has been reported yet (it's on my big backlog).

I'm not sure I have a useful recommendation here. If we add warnings, the page will become more cluttered, and it would also be Fedora specific (so it'd need to be a setting since Bodhi is used by other organizations too). I also don't think it'd be correct to encode a delay because some mirrors get the data very quickly, and others can lag by days.
